core/modules/replica/tools/qserv-replica-job-chunks.cc

```diff
@@ -1,6 +1,6 @@
 /*
  * LSST Data Management System
- * Copyright 2017 LSST Corporation.
+ * Copyright 2018 LSST Corporation.
```

Comment:
I think it should be 2017-2018. And we need to get rid of this nonsense altogether.

Reply:
I guess there is a plan to put the `Copyright` stuff into a separate file, one per directory. This needs to be the Qserv-wide refactoring.